eq-author/cypress/integration/e2e/e2e_spec.js

```diff
@@ -61,18 +62,22 @@ describe("End to end", () => {
   });
 
   it("Publishes successfully", () => {
-    cy.request(`http://localhost:19000/publish/${questionnaireId}`).then(
-      response => {
-        // Use this to grab a copy of what was sent from publisher
-        // cy.writeFile(
-        //   "cypress/fixtures/publisher_new.json",
-        //   JSON.stringify(response.body, null, 2)
-        // );
-        assertSecurityHeaders(response);
-        cy.fixture("publisher.json").then(expectedBody => {
-          expect(response.body).to.deep.equal(expectedBody);
-        });
-      }
-    );
+    const fieldsToIgnore = ["id", "form_type", "eq_id", "survey_id"];
+    cy.request(
+      `${Cypress.env("PUBLISHER_URL") ||
+        "http://localhost:9000"}/publish/${questionnaireId}`
+    ).then(response => {
+      // Use this to grab a copy of what was sent from publisher
+      // cy.writeFile(
+      //   "cypress/fixtures/publisher_new.json",
+      //   JSON.stringify(response.body, null, 2)
+      // );
```

Comment:
Should we just delete this code?

Reply:
Its left so when people need to update the expected output the code is there to do it. Basically to make it as easy as possible to keep adding things to the build questionnaire and then not having to worry about this bit. I found it non-obvious that you had to use the cypress methods and not jut normal node methods.